package.json

```diff
@@ -90,7 +90,7 @@
     "react-dom": "^16.4.1",
     "react-test-renderer": "^16.4.1",
     "semantic-release": "^12.4.1",
-    "styled-components": "^3.3.3",
+    "styled-components": "^4.0.0",
```

Comment:
This is not a peer dependency?

Reply:
Good catch, I think we will fix in a little bit in a separate pull (this one is merged now so oops)